index.js

```diff
@@ -3,6 +3,7 @@ const fs = require('fs');
 const path = require('path');
 const YAML = require('yamljs');
 const uniq = require('lodash/uniq');
+const _ = require('lodash');
```

Comment:
since we're importing the whole pkg you can combine this and line 5 and update line and update its use on line 225 (or add `isEmpty` to the import on l5)